api/src/main/kotlin/com/gmtkgamejam/models/Team.kt

```diff
@@ -9,7 +9,9 @@ data class Team (
     var author: String,
     var authorId: String,
     var description: String,
+
     var skillsetMask: Int,
+    var skills: List<Skills>,
```

Comment:
Adding this as a parameter means it'll be automatically serialised, which we could potentially use for the deprecation of the bitwise ID.

If not, it's still handy for debugging.